WordPress/Classes/ViewRelated/Blog/EditSiteViewController.m

```diff
@@ -609,42 +708,34 @@ - (void)save:(id)sender
     [self.urlTextField resignFirstResponder];
     [self.usernameTextField resignFirstResponder];
     [self.passwordTextField resignFirstResponder];
-
     self.url = [NSURL IDNEncodedURL:self.urlTextField.text];
-    DDLogInfo(@"blog url: %@", self.url);
     self.username = self.usernameTextField.text;
     self.password = self.passwordTextField.text;
-
-    if (!self.savingIndicator) {
-        self.savingIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
-        self.savingIndicator.autoresizingMask = UIViewAutoresizingFlexibleLeftMargin | UIViewAutoresizingFlexibleRightMargin;
-        CGRect frm = self.savingIndicator.frame;
-        frm.origin.x = (self.tableView.frame.size.width / 2.0f) - (frm.size.width / 2.0f);
-        self.savingIndicator.frame = frm;
-        UIView *aView = [[UIView alloc] initWithFrame:CGRectMake(0.0f, 0.0f, self.tableView.frame.size.width, frm.size.height)];
-        [aView addSubview:self.savingIndicator];
-
-        [self.tableView setTableFooterView:aView];
-    }
-    [self.savingIndicator setHidden:NO];
-    [self.savingIndicator startAnimating];
-
-    if (self.blog) {
-        self.blog.geolocationEnabled = self.geolocationEnabled;
-        [[ContextManager sharedInstance] saveContext:self.blog.managedObjectContext];
-    }
-    if (self.blog == nil || self.blog.usernameForSite == nil) {
-        [self validateUrl];
-    } else {
-        if ([self.startingUser isEqualToString:self.usernameTextField.text] &&
-            [self.startingPwd isEqualToString:self.passwordTextField.text] &&
-            [self.startingUrl isEqualToString:self.urlTextField.text]) {
-            // No need to check if nothing changed
-            [self cancel:nil];
-        } else {
-            [self validateUrl];
-        }
+    BlogService *blogService = [[BlogService alloc] initWithManagedObjectContext:self.blog.managedObjectContext];
+    self.blog.blogName = self.siteTitleTextField.text;
+    self.blog.blogTagline = self.siteTaglineTextField.text;
+    self.blog.geolocationEnabled = self.geolocationEnabled;
+    if ([self.blog hasChanges]) {
+        [SVProgressHUD show];
+        [blogService updateSettingForBlog:self.blog success:^{
+            [SVProgressHUD showInfoWithStatus:@"Settings Saved"];
+        } failure:^(NSError *error) {
+            [SVProgressHUD showErrorWithStatus:@"Settings update failed"];
+        }];
     }
+    
+//    if (self.blog == nil || self.blog.usernameForSite == nil) {
+//        [self validateUrl];
+//    } else {
+//        if ([self.startingUser isEqualToString:self.usernameTextField.text] &&
+//            [self.startingPwd isEqualToString:self.passwordTextField.text] &&
+//            [self.startingUrl isEqualToString:self.urlTextField.text]) {
+//            // No need to check if nothing changed
+//            [self cancel:nil];
+//        } else {
+//            [self validateUrl];
+//        }
+//    }
```

Comment:
Don't commit commented out code

https://github.com/wordpress-mobile/WordPress-iOS/wiki/WordPress-for-iOS-Objective-C-Style-Guide#before-you-commit